Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -119,20 +119,24 @@ class CreateSafeFormUIModel {
     }
 
     var isCreateEnabled: Bool {
-        (state == .ready || (state == .keyNotFound && userSelectedPaymentMethod == .relayer)) &&
+        (state == .ready || (state == .keyNotFound && userSelectedRelayerAndItIsAvailableForChain)) &&
         name != nil &&
         !name!.isEmpty &&
         chain != nil &&
         !owners.isEmpty &&
         threshold > 0 &&
         threshold <= owners.count &&
-        (selectedKey != nil || userSelectedPaymentMethod == .relayer) &&
+        (selectedKey != nil || userSelectedRelayerAndItIsAvailableForChain) &&
         transaction != nil &&
-        (deployerBalance != nil || userSelectedPaymentMethod == .relayer) &&
-        (deployerBalance! >= transaction.requiredBalance || userSelectedPaymentMethod == .relayer) &&
+        (deployerBalance != nil || userSelectedRelayerAndItIsAvailableForChain) &&
+        (deployerBalance! >= transaction.requiredBalance || userSelectedRelayerAndItIsAvailableForChain) &&
         error == nil
     }
 
+    var userSelectedRelayerAndItIsAvailableForChain: Bool {
```

Comment:
Maybe we can find a shorter name? Any suggestions?

Reply:
user should not be able to select a relayer if it's not available for the chain

Reply:
+1